jenkins/aws/buildSwagger.sh

```diff
@@ -60,18 +60,24 @@ SWAGGER_SPEC_YAML_FILE=$(findFile \
 #                    "${AUTOMATION_BUILD_DEVOPS_DIR}/openapi_extensions.yaml" \
 #                    "${AUTOMATION_BUILD_DEVOPS_DIR}/codeontap/openapi_extensions.yaml")
 
+
+# Collect all the possible files making up the spec into a directory tree
+# Files could be anywhere in the repo
+SWAGGER_BUNDLE_DIR="${tmpdir}/bundle"
+mkdir "${SWAGGER_BUNDLE_DIR}"
+pushd "${AUTOMATION_DATA_DIR}" > /dev/null 2>&1
```

Comment:
Should the be BUILD_SRC_DIR? AUTOMATION_DATA_DIR could include the devops JSON data files wouldn't it?

Reply:
Trouble is that the structure for ICS sees files to be included being in directories outside the scope of the BUILD_SRC_DIR. This is so they can become their own build dirs in the future. 

Thus the bundle process needs to reference files elsewhere in the repo, and the only  catch all way to do that would seem to be to go the top. I initially tried limiting the files to anything with "swagger" or "openapi" in it but then the devs broke that so I just figured I'd limit the bundle tree to .yaml and .json files.

Reply:
It means you may get quite a few extraneous files but it doesn't hurt.

Reply:
Maybe this should be a variable rather than a set value. WE do a similar thing for the docker context dir 

https://github.com/codeontap/gen3-automation/blob/85387f4711ad36fbf6289db82b8e0aac4b03d542/jenkins/aws/manageDocker.sh#L57 

Feels like using the default search everyone could end up with some unexpected behaviour. LIke merging of API's if there are two in the repo

Reply:
The starting point is always the spec file found via the BUILD_SRC_DIR so shouldn't get unexpected stuff included. Happy to add a variable if you feel more comfortable with that.